src/webViews/tools/dataExport.webview.ts

```diff
@@ -0,0 +1,329 @@
+            let selectedScopes = [];
+            let allScopeCnt = allScopes.length;
+            for(let i=0;i<allScopeCnt;i++){
+                if(allScopes[i].selected === true){
+                    if(i===0){
+                        selectedScopes.push("All Scopes");
+                    } else{
+                        selectedScopes.push(scopesSpecData[i-1]); // scopeSpecData is i-1 for eliminating all scopes
+                    } 
+                }
+            }
+            if(selectedScopes.includes("All Scopes")){
+                // No Collections Required
+                document.getElementById('collections').setAttribute('disabled',"");
+
+            } else {
+                const collectionsDropdown = document.getElementById('collections');
+                collectionsDropdown.innerHTML = '';
+                selectedScopes.map((scopeDetails)=>{
+                    // All Collections of the scope
+                    const selectAllOption = document.createElement('option');
+                    selectAllOption.value = 'All '+scopeDetails.name;
+                    selectAllOption.text = 'Select all collections of scope: '+scopeDetails.name;
+                    collectionsDropdown.appendChild(selectAllOption);
+
+                    // Individual Collections
+                    for(let collection of scopeDetails.collections){
+                        const collectionOption = document.createElement('option');
+                        collectionOption.value = collection.scopeName + "." + collection.name;
+                        collectionOption.text = collection.scopeName + "." + collection.name;
+                        collectionsDropdown.appendChild(collectionOption);
+                    }
+                    
+                })
+                document.getElementById('collections').removeAttribute('disabled');
+            }
+        }
+
+        $(document).ready(function() {
+            // Initialize Select2 on all dropdowns
+            $('.js-select2').select2();
+        });
+
+        window.addEventListener('message', event => {
+            const message = event.data; // The JSON data our extension sent
+        
+            switch (message.command) {
+                case 'vscode-couchbase.tools.dataExport.scopesInfo':
+                    let scopesData = message.scopes;
+                    scopesSpecData = scopesData;
+                    const scopeDropdown = document.getElementById('scopes');
+        
+                    // Clear existing options in the scope dropdown
+                    scopeDropdown.innerHTML = '';
+                    const selectAllOption = document.createElement('option');
+                    selectAllOption.value = 'All Scopes';
+                    selectAllOption.text = 'Select All';
+                    scopeDropdown.appendChild(selectAllOption);
+                    
+                    // Add scope options
+                    scopesData.forEach((scope) => {
```

Comment:
Change the icon from couchbase icon to any suitable icon

Reply:
Could you please change button color to Couchbase Red color

Reply:
done